### DISCUSSION TITLE
[FE]  Recoilの使用

### DISCUSSION THREAD

**Discussion by Rinasham** [Hackathon5-TeamD/evolution-quest#31] [Ideas]
ReactでContextの代わりにRecoilを使用したらproviderらへんがシンプルになるのかな？と思ったのですがどうでしょう？
https://zenn.dev/yuki_yuki/articles/c3bac6943f70b6

set関数のみのコンポーネントは再レンダリングも勝手にいい感じにしてくれるらしいので便利なのでは？(`·⊝·´)

(ディスカッション機能使ってみたかったので試しに書いてみてますw)

**Comment by haru-24** [comment]
Recoil便利そうですよね〜
いいと思います！

**Reply by Rinasham** [reply]
haru師匠にいいですねって言ってもらったんで使用決定です小林さん。w

**Reply by kobakichi** [reply]
まだ全然理解に至っていませんが、承知しました！！！！
